hikari/impl/rest.py

```diff
@@ -1997,6 +1997,21 @@ def fetch_members(
             entity_factory=self._entity_factory, request_call=self._request, guild=guild
         )
 
+    async def search_members(
+        self,
+        guild: snowflakes.SnowflakeishOr[guilds.PartialGuild],
+        name: str,
+    ) -> typing.Sequence[guilds.Member]:
+        route = routes.GET_GUILD_MEMBERS_SEARCH.compile(guild=guild)
+        query = data_binding.StringMapBuilder()
+        query.put("query", name)
+        query.put("limit", 1000)
```

Comment:
Shouldnt we also allow the user to pass the limit?

Reply:
we don't allow this for any "limited" endpoints right now and to allow it here seems inconsistent

Reply:
All the other endpoints are chunked, this is the only one that isnt (by the looks of it). So I don't see why not

Reply:
plus we don't have any precedent for specifying defaults for meta fields like that so either we inconsistently make the actual default for it a part of the interface which would be inconsistent with how we currently handle defaults or we make it implementation detail essentially making the default useless as it may be anything from 1 to 1000 dependent on implementation detail